boards/microbit_v2/src/main.rs

```diff
@@ -71,7 +71,7 @@ static mut CHIP: Option<&'static nrf52833::chip::NRF52<Nrf52833DefaultPeripheral
 /// Dummy buffer that causes the linker to reserve enough space for the stack.
 #[no_mangle]
 #[link_section = ".stack_buffer"]
-pub static mut STACK_MEMORY: [u8; 0x1500] = [0; 0x1500];
```

Comment:
Why did this need to increase?

Reply:
It faults with Kernel stack overflow otherweise